src/main/java/org/retest/rebazer/connector/GithubConnector.java

```diff
@@ -70,6 +70,8 @@ public void merge( final PullRequest pullRequest ) {
 		request.put( "merge_method", "merge" );
 
 		template.put( requestPath( pullRequest ) + "/merge", request, Object.class );
+
+		template.delete( "/git/refs/heads/" + pullRequest.getSource() );
```

Comment:
Is merging executed asynchronously? If yes, is deletion executed afterwards or will there be conflicts if deletion is faster?

Reply:
It's synchronously